app/src/main/res/layout-v21/login.xml

```diff
@@ -140,6 +140,18 @@
             android:visibility="gone" />
 
         <TextView
+            android:id="@+id/register"
+            android:layout_width="wrap_content"
```

Comment:
@vishwesh3 You can style it in the same way and sequence , Like the legal info and another server textViews. :)